.flowconfig

```diff
@@ -23,5 +23,6 @@ module.name_mapper.extension='md' -> '<PROJECT_ROOT>/flow/WebpackAsset.js.flow'
 module.name_mapper='^baseui' ->'<PROJECT_ROOT>/src'
 module.name_mapper='^examples' ->'<PROJECT_ROOT>/documentation-site/examples'
 log.file=.flow.log
+sharedmemory.hash_table_pow=21
```

Comment:
why is this needed?

Reply:
I am not sure :/ the storybook maintainers suggested this